src/main/scala/recfun/Main.scala

```diff
@@ -0,0 +1,49 @@
+package recfun
+
+import scala.annotation.tailrec
+
+object Main {
+
+  def main(args: Array[String]) {
+    println("Pascal's Triangle")
+    for (row <- 0 to 10) {
```

Comment:
You can use `foreach`:

```suggestion
    (0 to 10).foreach { row =>
      (0 to row).foreach(print(pascal(_, row) + " ")
      println
    }
```

Even better, try to create function `def formatPascalTriangle(n: Int): String` where `n` represents number of rows. It's a more functional style.